src/match_parser.js

```diff
@@ -13,6 +13,9 @@ const wwwRegex = /(?:www\.)/;
 // anything looking at all like a domain, non-unicode domains, not ending in a period
 const domainNameRegex = /[A-Za-z0-9.-]*[A-Za-z0-9-]/;
 
+// anything looking like an email address
+const mailboxRegex = /[a-zA-z0-9.-]*@/;
```

Comment:
This seems like it might be a bit simplistic of a regex -- it wouldn't catch email addresses that include accent characters for example. Is it possible to use the JS example from https://emailregex.com?

Reply:
Sure! I'll make the change.

Reply:
Done. Only needed a bit of tweaking to get rid of the capturing groups since the current capture is strict about match ordering.